packages/myst-jb-toc/.gitignore

```diff
@@ -0,0 +1 @@
+licenses.json
```

Comment:
Can probably bring this up to the base gitignore?

Reply:
Perhaps? I think I added this through copying another package - let me confirm whether it's necessary.

Reply:
Ah, yes, that is from `myst-frontmatter` (when SPDX licenses update, we generate that json file) -- should be deleted here!